format/FormatCBFMiniPilatusDLS6MSN100.py

```diff
@@ -15,137 +15,6 @@
 from dxtbx.format.FormatCBFMiniPilatus import FormatCBFMiniPilatus
 from dxtbx.masking import GoniometerMaskerFactory
 
-# Module positional offsets in x, y, in pixels - for the moment ignoring the
```

Comment:
This is absolutely fine, but for reference these were drawn from the original DECTRIS metrology tables (probably obsolete anyways) but more adding a comment as to _why_ they were there in the first place. 

Removing almost certainly a good thing.